src/resolve/resolved-dns-cache.c

```diff
@@ -178,18 +174,21 @@ static void dns_cache_make_space(DnsCache *c, unsigned add) {
                 return;
 
         /* Makes space for n new entries. Note that we actually allow
-         * the cache to grow beyond CACHE_MAX, but only when we shall
-         * add more RRs to the cache than CACHE_MAX at once. In that
+         * the cache to grow beyond max_size, but only when we shall
+         * add more RRs to the cache than max_size at once. In that
          * case the cache will be emptied completely otherwise. */
```

Comment:
When you touch the comment, please rebreak lines as nowadays we allow 109ch per line.